dbtwiz/helpers/editor.py

```diff
@@ -14,5 +14,7 @@ def open_in_editor(path: Path) -> int:
         command = f"{editor} {path}"
     value = os.system(command)
     if value != 0:
-        warn(f"Failed to open file in editor. Command '{command}' returned status {value}.")
+        warn(
+            f"Failed to open file in editor. Command '{command}' returned status {value}."
+        )
```

Comment:
Dette er en type formattering jeg ikke er spesielt entusiastisk over. En lang linje til 3 linjer der den ene fortsatt er nøyaktig like lang...

Reply:
Possible alternative:
```
        warn(
            "Failed to open file in editor. "
            f"Command '{command}' "
            f"returned status {value}."
        )
```

Reply:
Changed to `warn(f"Failed to open file in editor. '{command}' returned status {value}.")`